src-tauri/src/main.rs

```diff
@@ -9,7 +9,7 @@ use std::io::{Read, Write};
 use std::path::Path;
 use std::process::{Command, Stdio};
 use std::sync::Mutex;
-use tauri::{Window, Manager};
+use tauri::{Manager, Window};
```

Comment:
this is interesting, either my format on save is not working correctly for imports or we are using different linters 🤔

Reply:
yeah it's me, format not working for some reason on save 👍